app/services/claim_service.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+require_dependency 'evss/claims_service'
+
+class ClaimService
+  EVSS_CLAIM_KEYS = %w(openClaims historicalClaims).freeze
+
+  def initialize(user)
+    @user = user
+  end
+
+  def all
+    raw_claims = client.all_claims.body
+    EVSS_CLAIM_KEYS.each_with_object([]) do |key, claims|
+      next unless raw_claims[key]
+      claims << raw_claims[key].map do |raw_claim|
+        find_or_initialize_claim(raw_claim)
+      end
+    end.flatten
+  rescue Faraday::Error::TimeoutError, Timeout::Error
```

Comment:
We can definitely tweak the timeout, currently it's just 10 seconds across the board. I would like to gather some data points before doing that though so we don't prematurely assume anything longer than 10 seconds is a timeout